cookiecutter.json

```diff
@@ -3,5 +3,7 @@
     "municipality_slug": "{{ cookiecutter.municipality.lower().replace(' ', '-').replace('_', '-') }}",
     "python_municipality_slug": "{{ cookiecutter.municipality_slug.replace('-', '_') }}",
     "maintainer_full_name": "Jackson Maxfield Brown",
-    "hosting_github_username_or_org": "CouncilDataProject"
+    "hosting_github_url": "https://github.com/CouncilDataProject/example",
+    "hosting_github_username_or_org": "{{ cookiecutter.hosting_github_url.replace('https://github.com/', '').split('/')[0] }}",
+    "hosting_github_reponame": "{{ cookiecutter.hosting_github_url.replace('https://github.com/', '').split('/')[1] }}"
```

Comment:
I think I would do this in the opposite order:

1. `hosting_github_username_or_organization`
2. `repository_name`
3. string combine to have `hosting_github_url`

Reply:
I went back and forth on how best to ask for this info. I like your suggestion @JacksonMaxfield ! I'll push an update later today.